lib/index.js

```diff
@@ -135,10 +135,11 @@ function build(babel) {
           var uid = this.path.parentPath.scope.generateUidIdentifierBasedOnNode(node.id);
           this.path.parentPath.scope.rename(node.id.name, uid.name);
           path.insertBefore([node]);
-          this.path.dangerouslyRemove();
+          this.path.remove();
         } else {
           var uid = path.scope.generateUidIdentifier("ref");
           var replacement = t.functionDeclaration(uid, node.params, normalizeFunctionBody(node.body));
+          replacement.loc = node.loc;
```

Comment:
I have my doubts about this change
It tested by `throw new Error('Wrong transformed function. maybe wrong sourcemap?');` in tests
I think that node.loc needed for source-maping, but I'm not sure